pkg/manager/member/tikv_member_manager.go

```diff
@@ -646,6 +646,11 @@ func (m *tikvMemberManager) syncTidbClusterStatus(tc *v1alpha1.TidbCluster, set
 	// This only returns Up/Down/Offline stores
 	storesInfo, err := pdCli.GetStores()
 	if err != nil {
+		if pdapi.IsTiKVNotBootstrappedError(err) {
+			klog.V(4).Infof("TiKV of Cluster %s/%s is not bootstrapped yet, get no store", tc.Namespace, tc.Name)
+			tc.Status.TiKV.Synced = true
+			return nil
```

Comment:
Should we return an error and stop syncing tikv here?

Reply:
If we return an error, the sync loop will be blocked here.